### PR TITLE
student profile now displays different message depending on completio…

### PR DESCRIPTION
**Pull request by emilia-friedberg**
…n of activity; activities are resumable

### CODE REVIEW

File: app/controllers/activity_sessions_controller.rb [hunk 1]
```diff
@@ -8,14 +8,23 @@ class ActivitySessionsController < ApplicationController
...
-       classroom_activity_id: @activity_session.classroom_activity_id,
-       is_retry: true
-      )
-      redirect_to play_activity_session_path(newSession)
+      started_session = ActivitySession.where(
+        user_id: @activity_session.user_id,
+        activity_id: @activity_session.activity_id
+        ).last
+      if started_session && started_session.started_at > @activity_session.completed_at
+        binding.pry
```
Comment (ddmck) [inline app/controllers/activity_sessions_controller.rb]: @emilia-friedberg you'll have to take out this binding before I can merge this.

File: app/controllers/activity_sessions_controller.rb [hunk 2]
```diff
@@ -8,14 +8,23 @@ class ActivitySessionsController < ApplicationController
...
+        redirect_to play_activity_session_path(started_session)
+      else
+        newSession = ActivitySession.create(
```
Comment (ddmck) [inline app/controllers/activity_sessions_controller.rb]: I know this is kind of pedantic and you didn't write it, but can you change this to new_session when you remove the binding.

File: app/controllers/activity_sessions_controller.rb [hunk 3]
```diff
@@ -8,14 +8,23 @@ class ActivitySessionsController < ApplicationController
...
+         state: "unstarted",
+         user_id: @activity_session.user_id,
+         activity_id: @activity_session.activity_id,
+         classroom_activity_id: @activity_session.classroom_activity_id,
+         is_retry: true
+        )
+        redirect_to play_activity_session_path(newSession)
```
Comment (ddmck) [inline app/controllers/activity_sessions_controller.rb]: this one too

File: app/controllers/activity_sessions_controller.rb (hunk 1 above)
Comment (emilia-friedberg) [inline app/controllers/activity_sessions_controller.rb]: bitmoji

**Comment by ddmck** [comment]
That was quick, thanks @emilia-friedberg !

**Pull request merged**